app/templates/administrative-units/administrative-unit/core-data/edit.hbs

```diff
@@ -230,6 +248,63 @@
           </:card>
         </EditCard>
 
+        <EditCard>
+          <:title>Gerelateerde organisaties</:title>
+          <:card as |Grid|>
+            {{#if
+              (eq
+                @model.administrativeUnit.constructor.modelName
+                "worship-service"
+              )
+            }}
+              <Grid as |Item|>
+                <Item>
+                  <:label>Centraal bestuur</:label>
+                  <:content>
+                    <CentralWorshipSelect
+                      @selected={{this.model.administrativeUnit.isSubOrganizationOf}}
+                      @onChange={{fn
+                        (mut this.model.administrativeUnit.isSubOrganizationOf)
+                      }}
+                    />
+                  </:content>
+                </Item>
+              </Grid>
+            {{/if}}
+            {{#if
+              (eq
+                @model.administrativeUnit.constructor.modelName
+                "central-worship-service"
+              )
+            }}
+              {{#each @model.administrativeUnit.subOrganizations as |worship|}}
+                <Grid as |Item|>
+                  <Item>
+                    {{! TODO: Enalble edit worship service }}
```

Comment:
Is that needed? Feels wrong to edit things like that on the edit page of a different thing?

Reply:
There isn't another place to edit this.

Reply:
Isn't a sub organization also an administrative unit?

Reply:
yes, what is your point?

Reply:
So why can't that just be edited on its own edit page?

Reply:
🤔 I mean to enable edit the link between central and worship service only.

Reply:
oooooooooooooooooooooooooooooooooh. right :sweat_smile: